src/modules/ROOT/pages/changelog/202209/09.json

```diff
@@ -87,8 +86,7 @@
     },
     {
       "what": [
-        "{APP_NAME_ADMIN} user view not properly updated after 'Confirm email address' was clicked in {APP_NAME_ADMIN}.",
-        "The user view was just staying empty, instead of being populated with the user data."
+        "{APP_NAME_ADMIN} user view not having any data present when 'Confirm email address' was clicked."
```

Comment:
`...view not having any data present when...` -> `...view not showing any data when...`

The original version reads a bit foo to me